res/linux/org.mixxx.Mixxx.metainfo.xml

```diff
@@ -7,6 +7,8 @@
   <project_license>GPL-2.0</project_license>
   <name>Mixxx DJ Software</name>
   <summary>Everything you need to perform live DJ mixes</summary>
+  <content_rating type="oars-1.1" />
```

Comment:
content_rating is optional. Why do we need to add it?

Reply:
Flathub builds fail without it.

Reply:
Then flathub is doing it wrong, forcing us to add a dummy value.

Reply:
Please add a comment that explains why this element is needed, although it is optional.

Reply:
It is not wrong. GNOME requires explicitly specifying that that there is no content rating to declare.